src/main/java/net/snowflake/client/jdbc/SnowflakeSQLLoggedException.java

```diff
@@ -93,25 +92,62 @@ private void sendOutOfBandTelemetryMessage(JSONObject value, SnowflakeSQLLoggedE
    * @param errorCode
    * @return JSONObject with data about SQLException
    */
-  private JSONObject createOOBValue(
+  static JSONObject createOOBValue(
       String queryId, String reason, String SQLState, int vendorCode, ErrorCode errorCode) {
     JSONObject oobValue = new JSONObject();
+    oobValue.put("type", TelemetryField.SQL_EXCEPTION.toString());
```

Comment:
Add driver type and version, slightly change field names to match with standard across drivers